app/src/main/java/eu/kanade/domain/manga/interactor/UpdateManga.kt

```diff
@@ -27,19 +30,27 @@ class UpdateManga(
     }
 
     suspend fun awaitUpdateFromSource(
+        source: Source,
```

Comment:
You don't need to pass the source here. The manga object has the source id use that to get the source instance in download manager